plugins/utils/spectre.nix

```
@@ -0,0 +1,282 @@
+{
+  lib,
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+with lib;
+  helpers.neovim-plugin.mkNeovimPlugin config {
+    name = "spectre";
+    originalName = "nvim-spectre";
+    defaultPackage = pkgs.vimPlugins.nvim-spectre;
+
+    maintainers = [maintainers.GaetanLepage];
+
+    description = ''
+      You may want to add your find/replace tool(s) to the `extraPackages` list:
+
+      ```nix
+        extraPackages = [pkgs.rg];
+      ```
```

```suggestion
      You may want to add your find/replace tool(s) to the `extraPackages` list:

      ```nix
        extraPackages = [pkgs.rg];
      ```

      or:

      ```nix
        plugins.spectre.findPackage = pkgs.rg;
        plugins.spectre.replacePackage = pkgs.gnused;
      ```
```

This description could be updated now that `{find,replace}Package` exist. Maybe even go further than my above suggestion and remove the old `extraPackages` stuff entirely?

---

You are definitely right. done